README.md

```diff
@@ -65,6 +65,12 @@ Import the default value from `pjax.esm.js` or `pjax.esm.min.js` as:
 import Pjax from './dist/pjax.esm.js';
 ```
 
+##### To import as ES module with Webpack 5 bundler
```

Comment:
This applies to not only webpack 5, but any tool that can parse the `exports` field well, so it's better if you widen that description. And import it this way is actually preferred, so the description could be like:

<pre>
# Pick a script in `dist` folder

## Automatic Pick

[[description]]<!-- e.g., If you are installing as a npm package, and the transpiler can parse the package `exports` field, try importing directly: -->

```js
import Pjax from '@sliphua/pjax';
```

## To declare globally

…

## To import as ES module

…
</pre>